src/main/java/flaxbeard/steamcraft/block/BlockPipe.java

```diff
@@ -334,10 +334,13 @@ public void registerBlockIcons(IIconRegister ir) {
     @SubscribeEvent
     public void onBlockHighlight(DrawBlockHighlightEvent event) {
         Item equipped = event.player.getCurrentEquippedItem() != null ? event.player.getCurrentEquippedItem().getItem() : null;
-        if ((event.target.typeOfHit == MovingObjectPosition.MovingObjectType.BLOCK) && (event.player.worldObj.getBlock(event.target.blockX, event.target.blockY, event.target.blockZ) instanceof BlockPipe)
-                && (event.player.getCurrentEquippedItem() != null) && ((event.player.getCurrentEquippedItem().getItem() instanceof IPipeWrench && ((IPipeWrench) equipped).canWrench(event.player, event.target.blockX, event.target.blockY, event.target.blockZ)))) {
-            RayTracer.retraceBlock(event.player.worldObj, event.player, event.target.blockX, event.target.blockY, event.target.blockZ);
-        }
+       if(equipped!=null){
```

Comment:
There actually already is a null check in the condition on the next couple lines.

A better solution for this would be to 1. store the Block in a variable so we don't have to call getBlock over and over again, and 2. use the equipped variable instead of calling getCurrentEquippedItem repeatedly.